client/src/pages/EditForm.js

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import TextField from "@mui/material/TextField";
+import { useState } from "react";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+const theme = createTheme();
+
+export default function EditForm() {
+  const [editing, setEditing] = useState(false);
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    console.log({
+      email: data.get("email"),
+      password: data.get("password"),
+    });
+  };
+
+  function isEditing(id) {
+    console.log("edit mode activated");
+    setEditing(id);
+  }
```

Comment:
This function is invoked when the user clicks the `Save` button below.  The function logs some helper text, sets `isEditing` to the value of the `id` passed into the function, and exits.  

Is this intentional?  Perhaps the "unit of work" here is to wire in the Save button and then a future PR will hook it up to the back end?

If that's the intent, it would be ideal to make that very obvious n the code.  You could add a `TODO` comment that lets readers of your code know that the remaining functionality is coming in a future PR.  Even removing the `isEditing(id)` call would help express your intent.  Right now it looks like a bug.

Speaking of the `isEditing(id)` call, that `id` is never passed into the function.  If you were to log it, you'd find the native JavaScript `event` object that came from clicking the `Save` button.  Even if you were to pass in a proper `id` here, are you sure that's necessary?  The `id` could come from the URL or be set on page load.  It doesn't need to be passed into the `onSave` handler, does it?

Reply:
Copy that and made necessary updates